NedyUdombat/Role-Assigner
# #167693845 Create middleware to validate request body

#### What does this PR do?
Creates a middleware to validate any request body

#### Description of Task to be completed?
- [x] create middleware
- [x] create schema for login request body
- [x] apply validator to auth routes

#### How should this be manually tested?
- Clone the repo,
- Run ```npm install``` to install all dependencies,
- Create a .env file using the .env.example as a guide and enter your environment variables
- Ensure you have a PostgreSQL server running on your development environment
- Run `sequelize db:create`  
- Run `sequelize db:migrate` 
- Run `sequelize db:seed:all`   
- Run npm run watch,
- On Postman, send a POST request `localhost:3000/api/v1/auth/login` without any body in your request
- The server response should correspond with the screenshot below
#### Any background context you want to provide?
- 
#### What are the relevant pivotal tracker stories?(if applicable)
[167693845](https://www.pivotaltracker.com/story/show/167693845)

#### Screenshots
<img width="474" alt="Screenshot 2019-08-05 at 2 05 14 AM" src="https://user-images.githubusercontent.com/12516195/62432082-7bc9b380-b725-11e9-979e-c57f687f930f.png">
